lib/date/DateRange.ts

```diff
@@ -83,26 +83,22 @@ export class FixedDateRange {
   }
 
   formattedTime () {
-    const start = dayjs(this.startDate)
     const end = dayjs(this.endDate)
-    const startDateFormat = formatTime(start)
+    const startDateFormat = this.formattedStartDate()
     const endDateFormat = formatTime(end)
 
     return `${startDateFormat} - ${endDateFormat}`
   }
-}
 
-const formatFromBasis = (basis: Dayjs, date: Dayjs) => {
-  const formattedTime = formatTime(date)
-  const current = now()
-  if (date.isToday()) return `Today ${formattedTime}`
-  if (date.isYesterday()) return `Yesterday ${formattedTime}`
-  if (date.isTomorrow()) return `Tomorrow ${formattedTime}`
-  if (date.isBetween(current, current.add(7, "days"), "days")) {
-    return date.format("ddd") + ` ${formattedTime}`
+  formattedStartTime () {
+    const start = dayjs(this.startDate)
+    return start.format("h:mm A")
+  }
+
+  formattedStartDate () {
+    const start = dayjs(this.startDate)
+    return formatDate(now(), start)
```

Comment:
We should probably just have general functions to format a single date and just call `formatDate(dateRange.startDate)` when we need to in specific areas. I think we have way too many of these formatter functions on the `FixedDateRange` class when it doesn't really make sense in a lot of cases.